### ISSUE TITLE
Inconsistency with Nomenclature `Emails` or `Email`

### ISSUE THREAD

**Issue by genericlady** [thoughtbot/bamboo#266]
In the README the module of `Emails` is given as an example.

```
defmodule MyApp.Emails do
  import Bamboo.Email

  def welcome_email do
    new_email(
      to: "john@gmail.com",
      from: "support@myapp.com",
      subject: "Welcome to the app.",
      html_body: "<strong>Thanks for joining!</strong>",
      text_body: "Thanks for joining!"
    )

    # or pipe using Bamboo.Email functions
    new_email
    |> to("foo@example.com")
    |> from("me@example.com")
    |> subject("Welcome!!!")
    |> html_body("<strong>Welcome</strong>")
    |> text_body("welcome")
  end
end
```

When in the docs for testing `Email` the singular form is used as an example.
https://hexdocs.pm/bamboo/Bamboo.Test.html
```
defmodule MyApp.Email do
  import Bamboo.Email

  def welcome_email(user) do
    new_email(
      from: "me@app.com",
      to: user,
      subject: "Welcome!",
      text_body: "Welcome to the app",
      html_body: "<strong>Welcome to the app</strong>"
    )
  end
end
```

Personally, I think the Singular form is better compared to Plural. Should the README be updated to `Email`?

**Comment by paulcsmith** [comment]
I don't have a strong preference either way, but you are 100% right that making them consistent would be better. Would you mind opening a PR to go with `Email`? If not LMK and I'll tackle it or add a label so someone else can give it a try.

Thanks for reporting this!

**Comment by genericlady** [comment]
@paulcsmith Thanks, I'd be happy to open a PR for it Friday. 👍

**Comment by genericlady** [comment]
@paulcsmith hey, I found this in the readme `MyApp.Register(user)` it seems like erlang syntax. Shouldn't it be something like `MyApp.Register.run(user)`.

**Comment by paulcsmith** [comment]
Closed by #270
